app/javascript/react/src/containers/Locations.js

```diff
@@ -17,29 +18,36 @@ class Locations extends Component {
 
   componentDidMount() {
     fetch('/api/v1/locations')
-    .then(response => {
+    .then(response => response.json())
+    .then(body => {
+      this.setState({ locations: body})
+
+    })
+
+      fetch('/api/v1/users',{
```

Comment:
Fix indentation here - otherwise it looks like the  second fetch call is nested.